app/scripts/controllers/permissions/enums.ts

```diff
@@ -2,4 +2,6 @@ export enum NOTIFICATION_NAMES {
   accountsChanged = 'metamask_accountsChanged',
   chainChanged = 'metamask_chainChanged',
   sessionChanged = 'wallet_sessionChanged',
+  walletNotify = 'wallet_notify',
+  solanaAccountChanged = 'solana_accountChanged',
```

Comment:
moving these 3 into the `@metamask/chain-agnostic-permission` package

Reply:
Here: https://github.com/MetaMask/core/pull/5491/files#diff-d4264b93bc87ab9ed17b687adfd862b58f089ff5a2430628205cb798aca271ddR127